viewer/package.json

```diff
@@ -32,7 +32,7 @@
     "test": "jest --maxWorkers=4",
     "test:visual:client": "cross-env JEST_PUPPETEER_CONFIG=visual-tests/jest-puppeteer.config.js jest --config=visual-tests/jest.config.js --verbose",
     "test:visual:server": "webpack-dev-server --config packages/webpack.config.js",
-    "test:visual": "concurrently --k --i --default-input-target 1 \"yarn run test:visual:server\" \"yarn run test:visual:client\"",
+    "test:visual": "concurrently -k -s \"first\" \"yarn run test:visual:client\" \"yarn run test:visual:server\"",
```

Comment:
No, the server doesn't actually ever exit, it just runs forever. Only reason it exits is the -k flag which says that when the first thing is done kill everything else. When killed it will get an exit code of 1. So if I chose "all" then this will always fail.